tensorflow/lite/python/lite.py

```diff
@@ -516,12 +516,17 @@ def convert(self):
 
     Raises:
       ValueError:
+        No concrete functions is specified.
```

Comment:
I would also suggest to add descriptions here about when it will raise `No concrete func` and when it will raise `multiple concrete func`.